src/olympia/abuse/management/commands/fill_cinderjobs_denormalized_fields.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+
+from olympia.abuse.models import CinderJob
+from olympia.addons.models import Addon
+
+
+class Command(BaseCommand):
+    help = (
+        'One-off command to fill CinderJob.target_addon and '
+        'CinderJob.resolvable_in_reviewer_tools fields'
+    )
+
+    def handle(self, *args, **options):
+        jobs = CinderJob.objects.all()
+
+        for job in jobs:
+            if abuse_report := job.initial_abuse_report:
+                job.target_addon = (
```

Comment:
Why is the logic here different than [here](https://github.com/mozilla/addons-server/pull/21882/files#diff-a1926bd1323775380703cbe2a00da0de6668413b9afab7815b337f5779d61c24R294-R298)

```python
target = abuse_report.target
job.target_addon = target is addon .... blah blah
```